shared/cmd/table.go

```diff
@@ -35,6 +36,10 @@ func RenderTable(format string, header []string, data [][]string, raw any) error
 		table.SetHeaderLine(false)
 		table.SetBorder(false)
 		table.Render()
+	case TableFormatSQLResult:
+		table := getBaseTable(header, data)
+		table.SetAutoFormatHeaders(false)
```

Comment:
why dont we want headers?

Reply:
This preserves the output format of `lxd sql` as it is. For example:
```
# Existing lxd sql format
$ lxd sql global 'select * from networks'
+----+------------+--------+-------------+-------+------+
| id | project_id |  name  | description | state | type |
+----+------------+--------+-------------+-------+------+
| 6  | 1          | lxdbr0 |             | 1     | 0    |
+----+------------+--------+-------------+-------+------+

# With auto formatted headers
$ lxd sql global 'select * from networks'
+----+------------+--------+-------------+-------+------+
| ID | PROJECT ID |  NAME  | DESCRIPTION | STATE | TYPE |
+----+------------+--------+-------------+-------+------+
| 6  | 1          | lxdbr0 |             | 1     | 0    |
+----+------------+--------+-------------+-------+------+
```

Reply:
cheers